js/screens/wizard-screen.js

```diff
@@ -2,7 +2,11 @@ import { initWizard } from '../wizard/wizard.js';
 
 export const wizardScreen = () => {
    const wizardScreenWrapper = document.createElement('div');
-   const wizard = initWizard(0);
+   wizardScreenWrapper.id = 'wizardScreenWrapper';
+   const wizard = initWizard({
+      wizardStep: 0,
+      characterIndex: 0,
```

Comment:
**issue:** Keeping selected character index is not a good idea, as the data order can change in time. I suggest you to assign **whole selected fighter object**, or fighter `id` - but then you'd have to add `id` to your data. Then, when you select another fighter, you just assign other object value

Reply:
This seems simple enough, but I'm not sure about accessing the data itself. Previously you said it's not a problem to have all of the character data stored as a file to be exported and shared around the functions as na argument, did I implement it wrong? When I tried to have it as a .JSON file, I was receiving some errors when trying to access it.